README.adoc

```diff
@@ -16,11 +16,20 @@ The following guides are available:
 * SUSE OpenStack Cloud, General:
 ** Security Guide
 ** Supplement to Administrator Guide and User Guide
+
+These guides are all maintained and created in this GitHub
+repository .
+
+The folowing guides are available upstream:
```

Comment:
* typo: fol**l**owing
* Maybe reword to "The following guides are built directly from upstream OpenStack sources:" to clarify what "available" means and what upstream you are talking about